src/BackpackServiceProvider.php

```diff
@@ -91,6 +91,10 @@ public function register()
             return new DatabaseSchema();
         });
 
+        $this->app->scoped('lifecycle-hook', function ($app) {
```

Comment:
Hmm seems a weird naming for this. If we are to follow the convention of the one below in `BackpackViewNamespaces` this should be:

```suggestion
        $this->app->scoped('BackpackLifecycleHooks', function ($app) {
```